docs/superpowers/plans/2026-07-04-release-please-migration.md

```
@@ -0,0 +1,950 @@
+- [ ] **Step 6: Commit**
+
+Run:
+
+```bash
+git add xtask/src/main.rs xtask/src/checks/release_versions.rs xtask/src/checks/release_versions_tests.rs CLAUDE.md Cargo.lock
+git commit -m "chore: remove git-cliff release fallback"
+```
+
+## Engineering Review: Applied Findings
+
+### Architecture
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)